install/docker/index.markdown

```diff
@@ -22,7 +21,7 @@ title: MoveIt 1 Docker Install
     </p>
     <br/>
     <p>
-      Note that for recent Linux distros, the installation is basically just a single wget command. You may also want <a href="https://docs.docker.com/install/linux/docker-ce/ubuntu/" target="_blank">add your user to the docker group</a> to avoid having to use sudo permissions when you use the docker command.
+      Note that for recent Linux distros, the installation is basically just a single <span class="ros-command">wget</span> command. You may also want <a href="https://docs.docker.com/install/linux/docker-ce/ubuntu/" target="_blank">add your user to the docker group</a> to avoid having to use sudo permissions when you use the docker command.
```

Comment:
Why using ros-command class and html-span tags?
```suggestion
      Note that for recent Linux distros, the installation is basically just a single `wget` command. You may also want <a href="https://docs.docker.com/install/linux/docker-ce/ubuntu/" target="_blank">add your user to the docker group</a> to avoid having to use sudo permissions when you use the docker command.
```

Reply:
We're using these <span> tags throughout this page. Unfortuantly this page is in HTML, not Markdown, but it does result in a nicer looking webpage. I don't think your suggestion makes sense.

Reply:
You are right. Just inferred from file ending `.markdown` that this should be markdown...